### ISSUE TITLE
Conception issue

### ISSUE THREAD

**Issue by anodynos** [sergeyzwezdin/Knockout-MultiModels#1]
I am sorry, but I think one doesn't quite understand how your proposal is effectively different from ko2.1's 

```
ko.applyBindings(myViewModel, document.getElementById('someElementId')). 
```

Could you please indicate a concrete scenario that can't be (easily) done with native ko whereas MultiModals shines ?

**Comment by abergs** [comment]
I agree. What is the difference? (edit: Realizing i sound like a douche, not intended)

**Comment by cdaly2012** [comment]
I completely get what this plugin does. I needed to avoid grouping my form fields on the page. This was exactly what I was looking for.

There was just one alteration I needed to make to get my requirement to work - the refreshBindings method was restricted to only allow new fields in and not update the existing fields. I removed this limitation and my form now has the flexible 2 way binding I needed using multiple ViewModels. My form has a reset button which gets the original view model back and updates the fields. 

Thanks!
